2024/BIN-2024-0003.md

```diff
@@ -1,122 +1,22 @@
 | BIN-2024-0003 | `CHECKSIGFROMSTACK`
 | :------------ | :-------
-| Revision      | 001 (2024-01-27)
+| Revision      | 002 (2025-08-19)
 | Author        | Brandon Black `<freedom@reardencode.com>`
 | |
 | Layer         | Consensus (soft fork)
 | Status        | Draft
 | License       | BSD-3-CLAUSE
 | |
 | Discussion    | https://delvingbitcoin.org/t/lnhance-bips-and-implementation/376/9
-| Aliases       | [BIPs PR#1535](https://github.com/bitcoin/bips/pull/1535)
+| Aliases       | [BIP-348](https://github.com/bitcoin/bips/bip-0348.mediawiki)
 
-## Abstract
+## Reference
 
-This BIP describes two new opcode for the purpose of checking cryptographic
-signatures in bitcoin scripts against data other than bitcoin transactions.
+This document assigns BIN-2024-0003 to BIP-348.
 
-## Summary
+## Revisions
 
-We propose replacing `OP_NOP5` (0xb4) in bitcoin script with
-`OP_CHECKSIGFROMSTACKVERIFY`. When verifying taproot script spends having
-leaf version 0xc0 (as defined in [BIP 342]), we propose `OP_CHECKSIGFROMSTACK`
-to replace `OP_SUCCESS204` (0xcc).
+| Revision | Commit hash
+| :------- | :----
+| BIN-2024-0003 | [4a8035e38f8ff8f8a3898993fd78d49ba3b586f4](https://github.com/bitcoin-inquisition/binana/blob/4a8035e38f8ff8f8a3898993fd78d49ba3b586f4/2024/BIN-2024-0003.md)
```

Comment:
I meant something like

```
| BIN-2024-0003.001 | [1dee48343649585d5a4b469edfc40a0187d819c2](https://github.com/bitcoin/bips/blob/1dee48343649585d5a4b469edfc40a0187d819c2/bip-0348.md)
```

ie, a permalink to the version you're specifying, and the revision code (`.001`) you're using to identify it (matching the `Revision:` header). If there's no substantive changes, there's no need to bump the revision. If there were substantive changes, then having `3.1` pointing to the binana permalink and `3.2` pointing to the bips repo permalink would make sense, but it would also be good to have a summary of what the changes were somewhere.

Using a permalink in the `Aliases` header matching the current revision also seems good.

Reply:
ok, tried again. I added a "Commentary" column to explain the major difference, to avoid cluttering the main sections.